app/controllers/accounts/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class Accounts::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
```

Comment:
Why do we need this file?

Reply:
All these Devise controller files were generated when i run the rails generate command to override the registration controller. i will make sure to remove it.